lib/roby/app.rb

```diff
@@ -2511,7 +2521,9 @@ def find_dirs(*dir_path)
             if dir_path.last.kind_of?(Hash)
                 options = dir_path.pop
             end
-            options = Kernel.validate_options(options || {}, :all, :order, :path)
+            options =
+                Kernel.validate_options(options || {}, :all, :order, :path,
+                                        prefer_local: false)
```

Comment:
I am unsure about the name of the variable, its more of a "respect the order of root_paths" instead of prefer local.

Reply:
Decided to use that and also support specific_last, as the logic makes sense for that too